app/services/users.js

```diff
@@ -5,3 +5,11 @@ exports.registerUser = user =>
   users
     .create(user)
     .catch(error => Promise.reject(errors.databaseError(`${error.name}: ${error.errors[0].message}`)));
+
+exports.getUserByEmail = email =>
+  users
+    .findOne({
+      where: { email },
+      attributes: ['id', 'name', 'lastName', 'password']
```

Comment:
Try adding another attribute: `raw: true`. This will return only the object without `dataValues` and other properties you don't need. Also, is useful in performance terms.